src/main/java/com/minecolonies/coremod/colony/Structures.java

```diff
@@ -165,6 +148,50 @@ private static void loadStyleMaps()
         }
     }
 
+
+    private static void loadStyleMapsJar()
```

Comment:
This needs to be rewritten. It might work it might not. Use MC Resource system to load files from the JAR. It is a proven system and works regardless of dev environment, wrapped starter, etc.

This will break if somebody else opens the jars.
Resources are loaded before anything else.